pkg/printers/internalversion/describe.go

```diff
@@ -3633,14 +3632,20 @@ func (list SortableVolumeMounts) Less(i, j int) bool {
 	return list[i].MountPath < list[j].MountPath
 }
 
-func versionedClientsetForDeployment(internalClient clientset.Interface) versionedclientset.Interface {
+// TODO: get rid of this and plumb the caller correctly
+func versionedClientset(internalClient clientset.Interface) versionedclientset.Interface {
```

Comment:
Remove the function if no one is using it. If it's still being used, update it to versionedCoreClientV1.